packages/plugin-cloudflare/src/tunnel.ts

```diff
@@ -107,7 +116,7 @@ class TunnelClient {
 
         outputDebug('Cloudflared tunnel crashed, restarting...')
         // wait 1 second before restarting the tunnel, to avoid rate limiting
-        await sleep(1)
+        if (!isUnitTest) await sleep(1)
```

Comment:
I tried, but since we have two timers (this one and the global timeout), If I disable timers then the global timeout is triggered and I can't test this case. Any suggestion? 🤔